gradle-plugin/openshift/src/main/java/org/eclipse/jkube/gradle/plugin/OpenShiftExtension.java

```diff
@@ -103,4 +105,12 @@ public boolean getProcessTemplatesLocallyOrDefault() {
   public boolean isSupportOAuthClients() {
     return true;
   }
+
+  public File getOpenShiftManifestOrDefault(JavaProject javaProject) {
+    return getOpenShiftManifest().getOrElse(javaProject.getOutputDirectory().toPath().resolve(DEFAULT_OPENSHIFT_MANIFEST).toFile());
+  }
+
+  public File getImageStreamManifestOrDefault(JavaProject javaProject) {
+    return getImageStreamManifest().getOrElse(javaProject.getBaseDirectory().toPath().resolve(Paths.get("build", javaProject.getArtifactId() + "-is.yml")).toFile());
```

Comment:
Shouldn't this be based on the `javaProject#getBuldDirectory` or something that doesn't have the `build` literal hard coded?